packages/reports/tests/integration/components/asset-owner-test.ts

```diff
@@ -0,0 +1,24 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import moment from 'moment';
+import { ReportUserArgs } from 'navi-reports/components/asset-owner';
+
+module('Integration | Component | asset-owner', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it describes updated user and updated time on reports header', async function (assert) {
+    const now = new Date();
+    const arg: ReportUserArgs = { user: 'test-user', updatedOn: now.toISOString().split('.')[0] };
```

Comment:
Can we hardcode this to something like `2022-01-25T14:32:12` and same with assert, something like `test-user - Last Modified on Jan 25, 2022 2:32 PM`